session_crypto.py

```
@@ -20,6 +20,8 @@
 import json
 import logging
 import re
+import ctypes
```

Ruff (0.11.9)

`ctypes` imported but unused

Remove unused import: `ctypes`

(F401)